packages/compass-web/sandbox/index.tsx

```diff
@@ -33,6 +33,7 @@ import {
   useAtlasClusterConnectionsList,
 } from './atlas-cluster-connections';
 import { SandboxAutoconnectProvider } from '../src/connection-storage';
+import { sandBoxTelemetry } from './sandbox-telemetry';
```

Comment:
For consistency with other code, sandbox is one word

```suggestion
import { sandboxTelemetry } from './sandbox-telemetry';
```